### PR TITLE
Calculator logic functions

### PR DESCRIPTION
**Pull request by sinansevgi**
This pull request has calculator logic files. 

- Calculate.js keeps a record of data between operations and decides which operation should be done.

- Operate.js perform mathematical operations.

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import ButtonPannel from './ButtonPannel';
 function App() {
   return (
     <>
-      <Display result={0} />
+      <Display />
       <ButtonPannel />
     </>
```
Comment (Elbie-em) [inline src/components/App.js:15]: - [x] I noticed that you did not add the calculate.js import. Kindly import calculate.js in your `App` component as required.

File: README.md [hunk 2]
```diff
@@ -63,7 +64,7 @@ You don’t have to ever use `eject`. The curated feature set is suitable for sm
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/sinansevgi/mycalculator/issues/).
 
 ## Show your support
```
Comment (Elbie-em) [inline README.md:70]: - [x] I noticed a mixture of `CSS `files with components, kindly create a folder to store your `CSS` files and import them where required.

**Review by Elbie-em — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED 🟡  ♻️

Hi @sinansevgi, you have really done a great job so far, before this milestone is approved, there are a few issues that you need to work on to proceed, but you are almost there! Check my comments below ⬇️  for more details

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Kindly submit another review request once done and please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> Elbie Moonga (Part-time TSE)

> Github (@Elbie-em)
> Slack (Elbie Moonga)
> Twitter ([@ElbieEm](https://twitter.com/ElbieEm))
> LinkedIn([Elbie Moonga](https://www.linkedin.com/in/elbiemoonga/))

**Review by rindrajosia — APPROVED** [review]
Hi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------

**Pull request merged**